generators/server/templates/src/main/java/package/config/_WebConfigurer.java

```diff
@@ -53,13 +54,13 @@ public void onStartup(ServletContext servletContext) throws ServletException {
         EnumSet<DispatcherType> disps = EnumSet.of(DispatcherType.REQUEST, DispatcherType.FORWARD, DispatcherType.ASYNC);<% if (clusteredHttpSession == 'hazelcast') { %>
         initClusteredHttpSessionFilter(servletContext, disps);<% } %>
         initMetrics(servletContext, disps);
-        if (env.acceptsProfiles(Constants.SPRING_PROFILE_PRODUCTION)) {
-            initCachingHttpHeadersFilter(servletContext, disps);
-            initStaticResourcesProductionFilter(servletContext, disps);
-        }<% if (devDatabaseType == 'h2Disk' || devDatabaseType == 'h2Memory') { %>
-        if (env.acceptsProfiles(Constants.SPRING_PROFILE_DEVELOPMENT)) {
-            initH2Console(servletContext);
-        }<% } %>
+        if (env.acceptsProfiles(Constants.SPRING_PROFILE_PRODUCTION)) {<% if (!skipClient) { %>
```

Comment:
So, in such a project, a single person (with access to npm, and therefore gulp) would set up the codebase, with pre-injected html file into a git repository, and other devs work only from that, without ever trying a production build or even running unit tests?

I have to admit I did not consider that scenario, and you are correct that it would suffer from a change like this.

I believe there would be ways to allow using chrome as IDE, but as it would involve a kind of reverse watch, I expect you wouldn't think it's ideal...

So fair enough, I think we agree to disagree about the cost/benefit analysis :-)

To me the disadvantages you raise weren't even on the radar, if I'm completely honest. But on the other hand maybe they weren't that obvious, as they were not raised in the original issue.